ui/app/services/wizard.js

```diff
@@ -75,6 +90,32 @@ export default Service.extend(DEFAULTS, {
     }
     stateKey += state;
     this.set(stateType, stateKey);
+    if (stateType === 'featureState') {
+      //only track progress if we are on the first step of the first feature
+      if (
+        this.getCompletedFeatures().length === 0 &&
+        this.featureMachineHistory === null &&
+        (state === 'idle' || state === 'wrap')
+      ) {
+        let newHistory = [state];
+        this.set('featureMachineHistory', newHistory);
+      } else {
+        if (this.featureMachineHistory) {
```

Comment:
Could we move this whole block up above line 94? It'd just eliminate one level of nesting.

Reply:
I just broke this out into its own function while writing tests because it makes sense